apps/nrm/behaviours/types-of-exploitation.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+module.exports = superclass => class extends superclass {
+
+  /**
+   * Check if at least one address field has been populated
+   * @param {*} req - request object
+   */
+  // eslint-disable-next-line complexity
+  isAtLeastOneOptionSelected(req) {
+    if (
+      req.form.values['types-of-exploitation-forced-to-work'] ||
+      req.form.values['types-of-exploitation-wages-taken'] ||
+      req.form.values['types-of-exploitation-forced-to-commit-fraud'] ||
+      req.form.values['types-of-exploitation-prostitution'] ||
+      req.form.values['types-of-exploitation-child-exploitation'] ||
+      req.form.values['types-of-exploitation-taken-somewhere'] ||
+      req.form.values['types-of-exploitation-forced-to-commit-crime'] ||
+      req.form.values['types-of-exploitation-organs-removed'] ||
+      req.form.values['types-of-exploitation-unpaid-household-work'] ||
+      req.form.values['types-of-exploitation-other']
+    ) {
+        return true;
+      }
+
+      return false;
+  }
+
+  validate(req, res, next) {
+    if (!this.isAtLeastOneOptionSelected(req)) {
```

Comment:
1. Do you really need to pass in `req`? can you not just pass in `req.form.values`

2. Would it be more appropriate to make isAtLeastOneOptionSelected a private method and remove `this`? I don't like using `this` keyword unless I have to, also it means you have to write unit tests for this.  It's a bit pointless to have this function here if you're not able to reuse it.  

You don't have to do the 2nd point it's just an opinion

Reply:
Done.